tests/tests/test_monitor_client.py

```diff
@@ -140,6 +145,10 @@ def assert_valid_alert(message, bcc, subject):
     assert message["Subject"].startswith(subject)
 
 
+def stop_all_monitoring(mender_device):
+    mender_device.run(f"rm -f {MONITOR_ENABLED_DIR}/*")
```

Comment:
nit: technically it is `disable_all_checks`
another nit: I leave it up to you to decide, I believe we should use monitorctl to disable the checks (that way we do not have to update the test in case we change some logic behind the disable/enable operations). I know it is higher effort, since we do not have --disable-all-checks in monitorctl. I also know that parts of this file use the rad `ln` logic, but those parts predate the monitorctl command :)

Reply:
> nit: technically it is `disable_all_checks`

I was trying to mirror the method that creates the checks, which is called `prepare_service_monitoring`. But okey, I'll rename it.

> another nit: I leave it up to you to decide, I believe we should use monitorctl to disable the checks (that way we do not have to update the test in case we change some logic behind the disable/enable operations). I know it is higher effort, since we do not have --disable-all-checks in monitorctl. I also know that parts of this file use the raw `ln` logic, but those parts predate the monitorctl command :)

Nice suggestion but I am sorry to pass. Basically I would need to call `mender-monitorctl list`, parse the "enabled" part (both the subsystems and the check name!), then call `mender-monitorctl disable ...` - which I think is too much work for a boy scout.

Reply:
> > nit: technically it is `disable_all_checks`
> 
> I was trying to mirror the method that creates the checks, which is called `prepare_service_monitoring`. But okey, I'll rename it.
> 
> > another nit: I leave it up to you to decide, I believe we should use monitorctl to disable the checks (that way we do not have to update the test in case we change some logic behind the disable/enable operations). I know it is higher effort, since we do not have --disable-all-checks in monitorctl. I also know that parts of this file use the raw `ln` logic, but those parts predate the monitorctl command :)
> 
> Nice suggestion but I am sorry to pass. Basically I would need to call `mender-monitorctl list`, parse the "enabled" part (both the subsystems and the check name!), then call `mender-monitorctl disable ...` - which I think is too much work for a boy scout.

I know. it is ok.

boy scout as in [here](https://github.com/mendersoftware/integration/pull/2123#discussion_r951152056) ? :)

Reply:
Almost :stuck_out_tongue_closed_eyes: Rather [this other one](https://www.oreilly.com/library/view/97-things-every/9780596809515/ch08.html).